tests/testsuite/main.rs

```diff
@@ -24,6 +24,7 @@ mod cargo_command;
 mod cargo_config;
 mod cargo_env_config;
 mod cargo_features;
+mod cargo_new;
```

Comment:
I will move other tests for the `new` command to this mod after this PR merging.

Reply:
@epage Do you believe it's necessary to migrate it, or can we retain it?

Reply:
Moving things around are under discussion, so don't worry about it